### ISSUE TITLE
Improve link text names/descriptions in footer

### ISSUE THREAD

**Issue by cstarcher** [TexasDigitalLibrary/Vireo#1662]
Relevant Components:

- [x] Home
- [ ] File Upload

Description
The links in the footer have inaccessible names. The link destination should be clear from the accessible name. For example, “send an email to the webmaster” is more descriptive than “webmaster.”

Current Code
```<a href=…>Webmaster</a>```

Solution
Make sure the link destination is explicit, if not in the displayed label, then in the accessibility tree.
Recommended Code
```<a href=… aria-label=”Send an email to the webmaster”>Webmaster</a>```
